### DISCUSSION TITLE
Expanded Alt Key Behavior and related UI Enhancements.

### DISCUSSION THREAD

**Discussion by JezzyBelle** [voxeltycoon/issues#758] [UI&UX]
Alt key could have a third key press which would allow for another option of what to display. (I may set one of the three key presses to show only my garage/depots for a quick visual) 
Some things related and benefits to this as follows: 
  
~ _City name_ could be basic in one view. Just name, no icon and no population. Just a name tag. Could be standard game option. (I rarely even look at the actual population and don't need the icon to be able to tell it is a city. 
~ _Train view_ could just show the train icon (without a background to the icon) instead of the actual cargos. Sometimes I want to easily see the flow but not the clutter of all the cargo being represented. 
~ _Check box for assets._ Deselect so they do not display in any view. For example, I may not want some warehouses to show their cargo qty. That way I can quickly see cargos that are accessible via truck and not just part of the chain. Another example: I may want to see only certain waypoint labels displayed. EX: I sometimes only care to see one certain ones I am focused on OR see those that I have set up as user created map labels and do not serve as waypoints..
^ Scrap that last bit on waypoints and give us another asset and a make it selectable to display called "Map Labels". (with a check box to display or not as well)
 
~

**Comment by kalmix88-hun** [comment]
I agree. There should be an option to have the game show only icons or only nameplates. I often just want to watch the flow of public transport, but the many station names obscure everything.
A small station icon and the number of passengers would be enough for a compact view.
